src/login/73-seat-late.rules.in

```diff
@@ -14,6 +14,4 @@ ENV{ID_SEAT}=="", IMPORT{parent}="ID_SEAT"
 
 ENV{ID_SEAT}!="", TAG+="$env{ID_SEAT}"
 
-TAG=="uaccess", ENV{MAJOR}!="", RUN{builtin}+="uaccess"
```

Comment:
hmm, I think it would be prettier not to split this up further. Instead, let's just generate this with m4 and conditionalize the line as needed. The stuff in sysusers.d/ is built using m4 (and there too a couple of lines disappear under some build conditions) and I think we should do the same here.